src/Shared/FrameworkLocationHelper.cs

```diff
@@ -289,6 +293,7 @@ internal static class FrameworkLocationHelper
                 dotNetFrameworkVersion471,
                 dotNetFrameworkVersion472,
                 dotNetFrameworkVersion48,
+                dotNetFrameworkVersion481,
```

Comment:
Can you really target 4.8.1 from dev15 (this section)? Doesn't feel right to me.

Reply:
Not sure why I put this here in the first place, but I agree. There was another area where I created a spec for net481 -> VS 15, which I've changed to be net481->vs17